experimental/packages/exporter-trace-otlp-grpc/test/OTLPTraceExporter.test.ts

```diff
@@ -343,19 +342,19 @@ describe('when configuring via environment', () => {
   it('should use headers defined via env', () => {
     envSource.OTEL_EXPORTER_OTLP_HEADERS = 'foo=bar';
     const collectorExporter = new OTLPTraceExporter();
-    assert.deepStrictEqual(collectorExporter.metadata?.get('foo'), ['bar']);
+    assert.deepStrictEqual(collectorExporter['_metadata']?.get('foo'), ['bar']);
     envSource.OTEL_EXPORTER_OTLP_HEADERS = '';
   });
-  it('should override global headers config with signal headers defined via env', () => {
+  it.skip('should override global headers config with signal headers defined via env', () => {
```

Comment:
This test is skipped because I'm not sure if I agree with it in principle. @open-telemetry/javascript-maintainers

Reply:
I agree. I'll bring back the old behavior and we can make a new PR/issue to address it.

Reply:
I brought the old test back for now. I'll create an issue to update it.